src/category_theory/sites/spaces.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+
+import topology.opens
+import category_theory.sites.grothendieck
+import category_theory.sites.pretopology
+import category_theory.limits.lattice
+
+/-!
+# Grothendieck topology on a topological space
+
+Define the Grothendieck topology and the pretopology associated to a topological space, and show
+that the pretopology induces the topology.
+
+The covering (pre)sieves on `X` are those for which the union of domains contains `X`.
+
+## Tags
+
+site, Grothendieck topology, space
+
+## References
+
+* [https://ncatlab.org/nlab/show/Grothendieck+topology][nlab]
+* [S. MacLane, I. Moerdijk, *Sheaves in Geometry and Logic*][MM92]
+
+## Implementation notes
+
+We define the two separately, rather than defining the Grothendieck topology as that generated
+by the pretopology for the purpose of having nice definitional properties for the sieves.
+-/
+
+universe u
+
+variables (T : Type u) [topological_space T]
+
+namespace category_theory
+open topological_space limits
+
+/-- The Grothendieck topology associated to a topological space. -/
+def associated : grothendieck_topology (opens T) :=
```

Comment:
I think it makes more sense to call this `opens.grothendieck_topology` rather than `category_theory.associated`.

Reply:
Makes sense, I think I was going for `category_theory.grothendieck_topology.associated` but I got the namespaces wrong :) I prefer your suggestion anyway, and I'll use `opens.pretopology` for the analogous other version?